src/korc_hpc.f90

```diff
@@ -325,15 +325,23 @@ subroutine initialize_communications(params)
     endif
 
     !$OMP PARALLEL SHARED(params)
+#if defined(_OPENMP)
     params%num_omp_threads = OMP_GET_NUM_THREADS()
+#else
+    params%num_omp_threads = 1
+#endif
```

Comment:
What do you think about hiding at least the repeated `#ifdef`s inside functions?